matsim/src/main/java/org/matsim/core/mobsim/hermes/Hermes.java

```diff
@@ -35,28 +41,38 @@ final class Hermes implements Mobsim {
 	private ScenarioImporter scenarioImporter;
 	private final Scenario scenario;
 	private final EventsManager eventsManager;
+	private Map<String, TravelTime> travelTimes;
 
 	public Hermes(Scenario scenario, EventsManager eventsManager) {
 		this.scenario = scenario;
 		this.eventsManager = eventsManager;
+		Map<String, TravelTime> defaultTravelTimes = new HashMap<>();
+		for (String mode : this.scenario.getConfig().hermes().getMainModes()) {
+			defaultTravelTimes.put(mode, new DefaultTravelTime());
+		}
+		this.travelTimes = defaultTravelTimes;
```

Comment:
Is there a way to get at the travel times somehow, for unit testing? For example, I'm thinking of a test that modes all default to the default travel time when you make a new instance of the class, or that they use a non-default travel time for a mode after you've set it via `overrideTravelTimes`.